Problem3.py

```diff
@@ -0,0 +1,20 @@
+#Time complexity: O(n) where n is the number of characters in  the input string
+#Space complexity: O(n) where n is the number of characters in  the input string
```

Comment:
Since the input can only contain lowercase or uppercase alphabets, the space should be O(52) or O(1)